apps/client/src/routes/dashboards/dashboard/iot-app-kit-dashboard.d.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: Remove this file when the @iot-app-kit/dashboard declaration file is available.
+declare module '@iot-app-kit/dashboard';
```

Comment:
can we remove this? I think dashboard has it declaration at [here](https://github.com/awslabs/iot-app-kit/blob/main/packages/dashboard/package.json#LL22)

Reply:
@jmbuss fixed this in IoT App Kit and the new release should be out today to resolve.